backend/app/src/main/resources/db/migration/V1__Initial_ddl_script.sql

```diff
@@ -115,67 +105,96 @@ CREATE TABLE IF NOT EXISTS source_data (
-  hds_contact_name                      varchar(255),
-  hds_is_for_profit_flag                varchar(1),
-  source_status                         varchar(50),
-  hds_parent_ipc_id                     varchar(19),
-  bus_ipc_id                            varchar(19),
-  bus_cpn_id                            varchar(19),
-  bus_name                              varchar(100),
-  bus_legal_name                        varchar(255),
-  bus_payee_number                      varchar(255),
-  bus_owner_name                        varchar(255),
-  bus_owner_type                        varchar(255),
-  bus_owner_type_other                  varchar(255),
-  fac_building_name                     varchar(100),
-  facility_hds_details_additional_info  varchar(255),
-  physical_addr1                        varchar(255),
-  physical_addr2                        varchar(255),
-  physical_addr3                        varchar(255),
-  physical_addr4                        varchar(255),
-  physical_city                         varchar(255),
-  physical_province                     varchar(255),
-  physical_pcode                        varchar(255),
-  physical_country                      varchar(255),
-  phys_addr_is_private                  varchar(1),
-  mail_addr1                            varchar(255),
-  mail_addr2                            varchar(255),
-  mail_addr3                            varchar(255),
-  mail_addr4                            varchar(255),
-  mail_city                             varchar(255),
-  mail_bc                               varchar(255),
-  mail_pcode                            varchar(255),
-  mail_country                          varchar(255),
-  mail_addr_is_private                  varchar(1),
-  facility_id							bigint,
+  do_not_load_flag VARCHAR(10),
+  stakeholder VARCHAR(20),
+  stakeholder_id VARCHAR(19),
+  hds_ipc_id VARCHAR(19),
+  hds_cpn_id VARCHAR(19),
+  hds_provider_identifier1 VARCHAR(50),
+  hds_provider_identifier2 VARCHAR(50) ,
+  hds_provider_identifier3 VARCHAR(50),
+  hds_provider_identifier_type1  VARCHAR(50),
+  hds_provider_identifier_type2  VARCHAR(50),
+  hds_provider_identifier_type3  VARCHAR(50),
+  hds_msp_facility_number VARCHAR(50),
+  hds_pauth_id VARCHAR(30),
+  hds_category_code VARCHAR(15), 
+  hds_role_type_code VARCHAR(15), 
+  hds_type VARCHAR(400),
+  hds_sub_type VARCHAR(50),
+  hds_user_chid VARCHAR(50),
+  hds_created_dts VARCHAR(255),
+  hds_invalidated_dts VARCHAR(255),
+  hds_name VARCHAR(100),
+  hds_preferred_name_flag VARCHAR(1),
+  hds_email  VARCHAR(50),
+  hds_website  VARCHAR(50),
+  hds_bus_tel_area_code VARCHAR(15), 
+  hds_bus_tel_number VARCHAR(30), 
+  hds_tel_extension VARCHAR(15),
+  hds_cell_area_code VARCHAR(15), 
+  hds_cell_number VARCHAR(30),
+  hds_fax_area_code VARCHAR(15), 
+  hds_fax_number VARCHAR(30),
+  pcn_service_delivery_type VARCHAR(255),
+  pcn_clinic_type VARCHAR(255), 
+  pcn_pci_flag VARCHAR(1), 
+  hds_status VARCHAR(50) DEFAULT 'Active',
```

Comment:
I get concerned about default values in DB. It becomes less intuitive

Reply:
Agreed. I removed the default values from the database scripts.